deploy/cr.yaml

```diff
@@ -55,7 +55,12 @@ spec:
 #    ssl: my-cluster-name-ssl
 #    sslInternal: my-cluster-name-ssl-internal
 #    keyFile: my-cluster-name-mongodb-keyfile
-#    vault: my-cluster-name-vault
+#    vaultSpec:
```

Comment:
@pooknull please do not remove `vault: my-cluster-name-vault` operator uses it for data at rest encryption

Please use this structure: 
```
spec:
  vault:
    endpointUrl: https://vault-service:8200/          # Vault address
    tlsSecret: my-tls-vault-secret         # Kubernetes secret containing Vault TLS certs
    syncUsers:
      role: operator                                  # Vault Kubernetes role name
      mountPath: /secret/operator/psmdb/system
      tokenSecret: vault-token-operator # optional
```
Users should be able to use the service account or the secret